### DISCUSSION TITLE
Vessel log file

### DISCUSSION THREAD

**Discussion by OE1MWW** [jvde-github/AIS-catcher#233] [Q&A] [answered]
On the right side of the web view is an perfectly styled view for 'Ships'

Is there any command line switch, to store continously this data into a file ?
So, having a protocol, when and where the ships had been seen the first time.

**Comment by jvde-github** [comment, marked as answer]
Unfortunately not. One approach is to write the messages to a database (postgresql) or alternatively you can extract it from the webviewer, e.g. wget raspberrypi:8100/ships.json on linux and store this file every so many minutes with a structured filename.
But you have to write a script for that.

There are some plans to include this file dump in the server version (in CSV format) as a tool to quickly export to Postgres and to make the internal database persistent. Need to fix some other things before I can do that.

**Reply by OE1MWW** [reply]
Thanks for the reply. I will feed 'AISDecoder.exe' via UDP. This one writes a 'Vessels.dat', that is in fact, an .csv file.
Just want to use that setup during Transatlantic as a log, what have been heard ;-)
